### DISCUSSION TITLE
Mix of PCIe and M.2 corals?

### DISCUSSION THREAD

**Discussion by dz4news** [blakeblackshear/frigate#11088] [Ask A Question] [answered]
### What is your question:

I replaced my USB Coral with a mix of a PCIe and a M.2 coral. After struggling a few days to get the drivers installed, I have everything up and working.  Am I correct in my thinking that my current trained model works with the mixed corals? Everything seem fine.. 
Currently running on a bare metal - Dell Optiplex 7010 micro, Ubuntu 23.10, Docker Compose
<img width="1385" alt="Screenshot 2024-04-23 at 9 29 36 PM" src="https://github.com/blakeblackshear/frigate/assets/57462035/dfc9730d-dd9f-4b3d-ab64-2ee4381e05c5">

**Comment by NickM-27** [comment, marked as answer]
Yes, they process data internally the same
